test/smoke/generated/purview-administration-rest/README.md

```diff
@@ -0,0 +1,97 @@
+# Service client library for JavaScript
+
+Purview Account Client
+
+**Please rely heavily on [the service's documentation](https://azure.microsoft.com/en-us/services/purview/) and our [REST client docs](https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md) to use this library**
+
+Key links:
+
+- [Package (NPM)](https://www.npmjs.com/package/@msinternal/purview-administration-rest)
+- [Product documentation](https://azure.microsoft.com/en-us/services/purview/)
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 14.x.x or higher
+
+### Prerequisites
+
+- You must have an [Azure subscription](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell#authenticate-with-an-authentication-token) and follow [these](https://docs.microsoft.com/azure/purview/create-catalog-portal#add-a-security-principal-to-a-data-plane-role) instructions to use this package.
+
+### Install the `@msinternal/purview-administration-rest` package
+
+Install the Service client REST client library for JavaScript with `npm`:
+
+```bash
+npm install @msinternal/purview-administration-rest
+```
+
+### Create and authenticate a `PurviewAccount`
+
+To use an [Azure Active Directory (AAD) token credential](https://docs.microsoft.com/en-us/azure/cognitive-services/authentication?tabs=powershell#authenticate-with-an-authentication-token),
+provide an instance of the desired credential type obtained from the
+[@azure/identity](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) library.
+
+To authenticate with AAD, you must first `npm` install [`@azure/identity`](https://www.npmjs.com/package/@azure/identity)  and
+[enable AAD authentication on your Purview resource](https://docs.microsoft.com/azure/purview/create-catalog-portal#add-a-security-principal-to-a-data-plane-role)
+
+After setup, you can choose which type of [credential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) from `@azure/identity` to use.
+As an example, [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#defaultazurecredential)
+can be used to authenticate the client:
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
+AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET
+
+Use the returned token credential to authenticate the client:
+
+```typescript
+import PurviewAccount from "@azure-rest/purview-catalog";
```

Comment:
change the package name here. it should be `@msinternal/purview-administration-rest` in this case. As this package is a little special with multi-client configuration, there's no default export of PurviewAccount in this case. you will need to change it to 
```
import { PurviewAccountClient } from "@msinternal/purview-administration-rest";
```

Reply:
Updated